### DISCUSSION TITLE
Add query params for builds filtering

### DISCUSSION THREAD

**Discussion by marcocaldera** [codemagic-ci-cd/codemagic-docs#2405] [Ideas]
### Use case

Efficiently understanding how builds in every workflow are proceeding.

### Idea

Currently `https://codemagic.io/builds` is a big list of builds that merge all the workflow together.

If I want to understand the situation of a specific workflow I can filter it.
The problem is that there is no query param added to `https://codemagic.io/builds` which means that every time I want to check the situation of a workflow I need to perform multiple clicks and I cannot save a single bookmark and reuse it as a link.

For big projects with many builds, this would be highly appreciated.

**Comment by helinanever** [comment]
Hi @marcocaldera , thanks for this feature idea, we've taken note of it.
